docs/kyma/docs/042-gardener-kyma-deployment.md

```diff
@@ -0,0 +1,79 @@
+# Kyma on Gardener
+
+## Overview
+
+This walkthrough is designed to guide you step-by-step through the process of installing Kyma on Gardener. See the official [Gardener Github](https://github.com/gardener/gardener) page for more information regarding the project.
+
+The document provides the prerequisites and the instructions on how to install Kyma on a Gardener cluster.
+
+## Prerequisites
+
+- access to a Gardener project
+- a domain that you control
+
+>**NOTE:** This guide assumes you understand the [architecture of Gardener](https://github.com/gardener/documentation/wiki/Architecture) and the purpose of different cluster types.
+
+## Configure your Gardener project
+
+To configure your Gardener project, configure these items in the project dashboard:
```

Comment:
Well, this looks funny. 
Make it shorter?

Configure these items in the project dashboard: